draft-ietf-masque-connect-udp-listen.md

```diff
@@ -164,6 +166,36 @@ and therefore is to be ignored). This document does not define any parameters
 for the Connect-UDP-Bind header field value, but future documents might define
 parameters. Receivers MUST ignore unknown parameters.
 
+
+# The Proxy-Public-Address Response Header Field {#response}
+
+Upon accepting the request, the proxy MUST select at least one public IP
+address to bind. The proxy MAY also select a second address from a different address
+family. For each selected address, it MUST select an open port to
+bind to this request. From then and until the tunnel is closed, the proxy
+SHALL send packets received on these IP-port tuples to the client. The proxy
+then MUST communicate the selected addresses and ports to the client using
+the "Proxy-Public-Address" header. The format of that header is defined below
+using IPv4address, IPv6address and port from {{Section 3.2 of !URI=RFC3986}}.
+
+The proxy MUST then respond with the allocated IP and port pairings using the
+Proxy-Public-Address response header defined as follows:
+
+~~~ ascii-art
+proxy-public-address = IP-port-pairing (, IP-port-pairing)
+IP-port-pairing = ( IPv4address / "[" IPv6address "]" ) ":" port
+~~~
+{: #target-format title="Proxy Address Format"}
+
+If the proxy sends two IP-port pairings, they MUST be of different
+address families. If the client receives a Proxy-Public-Address field
```

Comment:
I think it's probably a mistake to limit this to one IP of each family.  There's nothing wrong with a proxy having multiple IPs of the same family on different network interfaces, in which case it would be optimal to allocate ports on all of them and allow ICE to find the best one.

Reply:
No strong opinion on doing this, but relaxing the limits feels like it would make parsing the field a whole lot easier.

Reply:
Thinking through this some more, maybe it's best to not restrict to two? I don't remember what the motivation was for the restriction